### PR TITLE
Removed explicit calls to get_rectangle_coordinates in draw_rectangle figure tests

### PR DESCRIPTION
**Pull request by jeffreypaul15**
### Description

Changed tests for draw_rectangle where the top_right coordinate was specified instead of using width and height ( removed repeated tests ).

### CODE REVIEW

File: sunpy/map/tests/test_plotting.py [hunk 1]
```diff
@@ -123,9 +123,8 @@ def test_rectangle_aia171_top_right(aia171_test_map):
     aia171_test_map.plot()
     bottom_left = SkyCoord(
         0 * u.arcsec, 0 * u.arcsec, frame=aia171_test_map.coordinate_frame)
-    w = 100 * u.arcsec
-    h = 100 * u.arcsec
-    bottom_left, top_right = get_rectangle_coordinates(bottom_left, width=w, height=h)
+    top_right = SkyCoord(
+        200 * u.arcsec, 500 * u.arcsec, frame=aia171_test_map.coordinate_frame)
```
Comment (ayshih) [inline sunpy/map/tests/test_plotting.py:127]:
```suggestion
        100 * u.arcsec, 100 * u.arcsec, frame=aia171_test_map.coordinate_frame)
```

File: sunpy/map/tests/test_plotting.py [hunk 2]
```diff
@@ -194,10 +193,8 @@ def test_heliographic_rectangle_top_right(heliographic_test_map):
     heliographic_test_map.plot()
     bottom_left = SkyCoord(
         60 * u.deg, 50 * u.deg, frame=heliographic_test_map.coordinate_frame)
-    w = 13 * u.deg
-    h = 13 * u.deg
-    bottom_left, top_right = get_rectangle_coordinates(bottom_left, width=w, height=h)
-    heliographic_test_map.draw_rectangle(bottom_left, width=w, height=h, color='cyan')
+    top_right = SkyCoord(80 * u.deg, 90 * u.deg, frame=hmap.coordinate_frame)
```
Comment (ayshih) [inline sunpy/map/tests/test_plotting.py:196]:
```suggestion
    top_right = SkyCoord(73 * u.deg, 63 * u.deg, frame=heliographic_test_map.coordinate_frame)
```

**Review by ayshih — CHANGES_REQUESTED** [review]
I like removing the calls to `get_rectangle_coordinates()`.

* If you keep the same rectangle sizes, you shouldn't even need to update the figure hashes.
* The changelog entry is not needed for such an internal change, so remove it.
* Remove the import of `get_rectangle_coordinates` at the beginning of the file.

**Review by ayshih — APPROVED** [review]
Looks good to me, squash when merging

**Review by nabobalis — APPROVED** [review]

**Pull request merged**